src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -55,6 +58,10 @@ final case class TenantApiServiceImpl(
 )(implicit ec: ExecutionContext)
     extends TenantApiService {
 
+  private val IPA: String = "IPA"
+  private val SAG: String = "SAG"
+  private val L37: String = "L37"
```

Comment:
What are those code referring to?

Reply:
These are constants needed to understand if Tenant Kind is IPA or GSP.  Maybe there is a best place for them?
Now these constants are here because are used only in this file.

Reply:
I'll begin giving them a different name, like `private val PUBLIC_ADMINISTRATIONS_IDENTIFIER: String = "IPA"` calling them `val x: string = "x"` doesn't carry all that info

Reply:
I agree, for reference:
- `L37` -> Gestori di Pubblici Servizi
- `SAG` -> Stazioni Appaltanti Gestori di Pubblici Servizi

Reply:
How about?
```suggestion
  private val PUBLIC_ADMINISTRATIONS_IDENTIFIER: String = "IPA"
  private val PUBLIC_SERVICES_MANAGERS: String = "SAG"
  private val CONTRACT_AUTHORITY_PUBLIC_SERVICES_MANAGERS: String = "L37"
```
maybe they're too verbose?

Reply:
Cool